MAUI/NumericEntry/Migration.md

```diff
@@ -0,0 +1,83 @@
+
+## Namespaces 
+
+<table>
+<tr>
+<th>Xamarin SfNumericTextBox <br> Xamarin SfNumericUpDown</th>
+<th>.NET MAUI SfNumericEntry</th></tr>
+<tr>
+<td>Syncfusion.SfNumericTextBox.XForms</td>
+<td>Syncfusion.Maui.Inputs</td></tr>
+</table>
+
+## Properties
+
+<table> 
+<tr>
+<th>Xamarin SfNumericTextBox <br> Xamarin SfNumericUpDown </th>
+<th>.NET MAUI SfNumericEntry</th>
+<th>Description</th></tr>
+<tr>
+<td>[BorderColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_BorderColor)</td>
+<td>[Stroke](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_Stroke)</td>
+<td>Gets or sets a color that describes the stroke.</td></tr>
+<tr>
+<td>[AllowDefaultDecimalDigits](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_AllowDefaultDecimalDigits)</td>
+<td>[AllowNull](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_AllowNull)</td>
+<td>Gets or sets a value indicating whether the SfNumericEntry allows null value or not.</td></tr>
+<tr>
+<td>[ClearButtonVisibility](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_ClearButtonVisibility)</td>
+<td>[ShowClearButton](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_ShowClearButton)</td>
+<td>Gets or sets a value indicating whether to show or hide the clear button in SfNumericEntry.</td></tr>
+<tr>
+<td>[FormatString](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_FormatString)</td>
+<td>[CustomFormat](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_CustomFormat)</td>
+<td>Gets or sets the format used to specify the formatting of Value.</td></tr>
+<tr>
+<td>[IsReadOnly](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_IsReadOnly)</td>
+<td>[IsEditable](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_IsEditable)</td>
+<td>Gets or sets a value indicating whether the SfNumericEntry allows editing the value or not.</td></tr>
```

Comment:
<td>Get or set a value indicating whether the SfNumericEntry allows editing the value or not.</td></tr>

Reply:
changes are done